packages/sdk/contractkit/src/wrappers/SortedOracles.test.ts

```diff
@@ -35,229 +41,321 @@ testWithGanache('SortedOracles Wrapper', (web3) => {
     }
 
     for (let i = 0; i < rates.length; i++) {
-      const tx = await sortedOracles.report(CeloContract.StableToken, rates[i], oracles[i])
+      const tx = await sortedOracles.report(target, rates[i], oracles[i])
       await tx.sendAndWaitForReceipt()
     }
   }
 
   // Quick setup for conditions when some oracle reports are expired and the rest are not.
   // This assumes that the rates reported can be arbitrary and not a critical piece of the test.
-  async function setupExpiredAndNotExpiredReports(expiredOracles: Address[]): Promise<void> {
+  async function setupExpiredAndNotExpiredReports(
+    sortedOracles: SortedOraclesWrapper,
+    target: ReportTarget,
+    expiredOracles: Address[],
+    allOracles: Address[]
+  ): Promise<void> {
     const expirySeconds = (await sortedOracles.reportExpirySeconds()).toNumber()
-    await reportAsOracles(expiredOracles)
+    await reportAsOracles(sortedOracles, target, expiredOracles)
     await timeTravel(expirySeconds + 5, web3)
-    const freshOracles = stableTokenOracles.filter((o) => !expiredOracles.includes(o))
-    await reportAsOracles(freshOracles)
+    const freshOracles = allOracles.filter((o) => !expiredOracles.includes(o))
+    await reportAsOracles(sortedOracles, target, freshOracles)
   }
 
+  /**
+   * When testing with a custom token pair we can't use the
+   * already deployed SortedOracles because that's managed
+   * by Governance and we can't execute changes on the contract.
+   * To make it easier we'll deploy an new version for use in
+   * the tests
+   */
+  async function newSortedOracles(owner: Address): Promise<SortedOraclesWrapper> {
+    const instance = await SortedOracles.new({ from: owner })
+    await instance.initialize(NetworkConfig.oracles.reportExpiry, { from: owner })
+    return new SortedOraclesWrapper(kit, instance.contract)
+  }
+
+  async function addOracleForPair(
+    sortedOraclesInstance: SortedOraclesWrapper,
+    identifier: CurrencyPairIdentifier,
+    oracle: Address,
+    owner: Address
+  ): Promise<void> {
+    // @ts-ignore
+    const sortedOraclesContract = sortedOraclesInstance.contract
+    await sortedOraclesContract.methods.addOracle(identifier, oracle).send({
+      from: owner,
+    })
+  }
+
+  // NOTE: These values are set in test-utils/network-config.json, and are derived
```

Comment:
nit: This file doesn't exist anymore, so the comment is misleading. New location is: `packages/dev-utils/src/migration-override.json`